src/dotnet/APIView/APIViewWeb/Controllers/AutoReviewController.cs

```diff
@@ -276,7 +276,7 @@ public async Task<ActionResult> CreateApiReview(
                         // But any manual pipeline run at release time should compare against all approved revisions to ensure hotfix release doesn't have API change
                         // If review surface doesn't match with any approved revisions then we will create new revision if it doesn't match pending latest revision
 
-                        bool considerPackageVersion = !String.IsNullOrWhiteSpace(codeFile.PackageVersion) && !(new AzureEngSemanticVersion(codeFile.PackageVersion, codeFile.Language).IsPrerelease);
+                        bool considerPackageVersion = !String.IsNullOrWhiteSpace(codeFile.PackageVersion);
```

Comment:
I am not familiar with this code :( but looking at the history, it seems it was intentional that Dozie added the part you are removing.
See: https://github.com/Azure/azure-sdk-tools/pull/10788

Is `packageVersion=null` no longer a thing?

Reply:
we are preserving the `packageVersion=null` but removing the `prerelease` that has been in there even longer!! but I think @praveenkuttappan  has some context if this is necessary or if we can do this change

Reply:
Thanks for the clarifications